parachains/runtimes/assets/statemine/src/weights/cumulus_pallet_xcmp_queue.rs

```diff
@@ -54,8 +54,8 @@ impl<T: frame_system::Config> cumulus_pallet_xcmp_queue::WeightInfo for WeightIn
 		//  Measured:  `76`
 		//  Estimated: `571`
 		// Minimum execution time: 4_956 nanoseconds.
-		Weight::from_ref_time(5_108_000)
-			.saturating_add(Weight::from_proof_size(571))
+		Weight::from_parts(5_108_000, 0)
+			.saturating_add(Weight::from_parts(0, 571))
```

Comment:
```suggestion
		Weight::from_parts(5_108_000, 571)
```

So many more instances of this... I think we need to reweigh everything with the new weights code?

Reply:
Yes, re-weighting happens before each release. It takes 6-8 hours AFAIK and so to get it merged sooner I replaced it in the code.